package-lock.json

```diff
@@ -1,14 +1,2211 @@
 {
     "name": "porter-vscode",
     "version": "0.0.3",
-    "lockfileVersion": 1,
+    "lockfileVersion": 2,
     "requires": true,
+    "packages": {
+        "": {
+            "name": "porter-vscode",
```

Comment:
I don't know npm at all but it seems weird that porter-vscode is listed as a dependency for porter-vscode and then all of the dependencies were re-added to the lockfile, nested under packages.

Does that seem right to you?

Reply:
☕️💡 Oh this is `package-lock.json` file which is automatically generated when `npm install` is run, I believe it must be reference of this name via `package.json` https://github.com/getporter/vscode-extension/blob/main/package.json#L2 

* More about `package-lock.json` - https://docs.npmjs.com/cli/v9/configuring-npm/package-lock-json